Mathlib/Tactic/TFAE.lean

```diff
@@ -67,12 +66,9 @@ of hypotheses of the form `Pᵢ → Pⱼ` or `Pᵢ ↔ Pⱼ` have been introduce
 Example:
 ```lean
 example : TFAE [P, Q, R] := by
-  tfae_have 1 → 2
-  · /- proof of P → Q -/
-  tfae_have 2 → 1
-  · /- proof of Q → P -/
-  tfae_have 2 ↔ 3
-  · /- proof of Q ↔ R -/
+  tfae_have 1 → 2 := sorry /- proof of P → Q -/
+  tfae_have 2 → 1 := sorry /- proof of Q → P -/
+  tfae_have 2 ↔ 3 := sorry /- proof of Q ↔ R -/
```

Comment:
I wonder if we want to write
```suggestion
  tfae_have 1 → 2 := sorry /- proof of P → Q -/
  tfae_have 2 → 1 := sorry /- proof of Q → P -/
  tfae_have 2 ↔ 3 := by
    sorry /- tactic proof of Q ↔ R -/
```
to make it clear how to use tactic here to the reader

Reply:
Maybe, but this is the docstring for `tfae_finish`, so I wanted to leave the `tfae_have` parts uncluttered just to demonstrate `tfae_finish`. The demonstration of different `tfae_have` syntax is in the `tfae_have` docstring. Also note that the tactic syntax is "mathlib have" style, not just `by`:
```lean
tfae_have 1 → 2
· admit
```

Reply:
Ah, I missed this was for `tfae_finish`.